frontend/src/scenes/plugins/source/PluginSource.tsx

```diff
@@ -107,6 +107,7 @@ export function PluginSource({
                         </p>
 
                         {pluginSourceLoading ? (
+                            // eslint-disable-next-line react/forbid-elements
```

Comment:
Skipping this one for now because I'm conscious it might be gone soon with @Twixes work on the new Apps UI